spec/models/participant_declaration/npq_spec.rb

```diff
@@ -8,6 +8,16 @@
       create_list(:npq_participant_declaration, 20)
     end
 
+    describe "associations" do
+      let(:user) { create(:user) }
```

Comment:
could we add a factory for outcomes?

Reply:
Done https://github.com/DFE-Digital/early-careers-framework/pull/2755/commits/85b96f3a568339e19b5dfb04c6c212d45fa737e1